packages/assets-controllers/src/AccountTrackerController.ts

```diff
@@ -264,14 +270,16 @@ export class AccountTrackerController extends StaticIntervalPollingControllerV1<
    * @param interval - Polling interval trigger a 'refresh'.
    */
   async poll(interval?: number): Promise<void> {
```

Comment:
This feels more of a cleanup task to me rather than connected to converting to BaseController v2 — what are your thoughts about doing that in another PR?